stack_master.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "table_print"
   spec.add_dependency "dotgpg"
   spec.add_dependency "deep_merge"
+  spec.add_dependency "cfndsl"
```

Comment:
Currently we have hard dependencies on sparkleformation and cfndsl, although some users of SM will never use them or the associated functionality. I wonder if we could not have a hard dependency on them, check for them if the user attempt to use them, then complain and fail if they don't have them (+ask them to install them first if they want to use whatever template they attempted to). Thoughts?